packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -220,6 +221,11 @@ class VideoEvent {
   /// Only used if [eventType] is [VideoEventType.initialized].
   final Size? size;
 
+  /// Radians to rotate the video so it is displayed correctly.
+  ///
+  /// Only used if [eventType] is [VideoEventType.initialized].
+  final double? rotationCorrection;
```

Comment:
Why use radians?  That seems inconvenient when most people will be using [90, 180, 270] degrees.

Reply:
Changed to degrees.

No on should need to set these values since it's set by the framework, but that's a good point. I think I originally used radians because that's what Transform.rotate accepts.